nucypher/blockchain/eth/clients.py

```diff
@@ -386,6 +386,9 @@ def get_blocktime(self):
         now = highest_block['timestamp']
         return now
 
+    def get_block(self, block_number):
+        return self.w3.eth.getBlock(block_number)
```

Comment:
The convention for `web3.eth.getBlock(...)` is that it can take a block number, or 'latest', or a hash, and takes a `block_identifier` as the parameter. We should do the same.

```suggestion
    def get_block(self, block_identifier):
        return self.w3.eth.getBlock(block_identifier)
```